src/classic.c

```diff
@@ -10,5 +10,6 @@ bool is_running_on_classic_distribution()
 	    || access("/var/lib/dpkg/status", F_OK) == 0
 	    || access("/var/lib/pacman", F_OK) == 0
 	    || access("/var/lib/portage", F_OK) == 0
-	    || access("/var/lib/rpm", F_OK) == 0;
+	    || access("/var/lib/rpm", F_OK) == 0
+	    || access("/sbin/procd", F_OK) == 0;
```

Comment:
the database is not mandatory, you can build all the daemons in the firmware instead of installing them at runtime